cfme/configure/access_control.py

```diff
@@ -1177,10 +1178,14 @@ class TenantForm(ConfigurationView):
     """ Tenant Form """
     name = Input(name='name')
     description = Input(name='description')
-
+    add_button = Button('Add')
     cancel_button = Button('Cancel')
 
 
+class ListEntity(BaseListEntity):
```

Comment:
**REQUIRED:** Why are we subclassing this if we are not adding anything to it? line 1221 can just be 
```list_entity = BaseListEntity```

Reply:
I used it that way itself, @quarckster required to use it like this
@psav what's the final change?

Reply:
@vatsalparekh, you shouldn't use ListEntity directly. Use BaseEntitiesView instead and switch view mode to List View.
So, get_entity() and other methods will return you ListEntity()

Reply:
I proposed to do that as it done [here](https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/host_views.py#L73), I was thinking that there are other kinds of entities like grid or tile. Also should we directly use Base entities?

Reply:
@quarckster, we can but I wouldn't recommend to do that.  I'll take a look what we can do there and come up with something in a bit.

Reply:
it is interesting idea to reuse ListEntity here. However, unfortunately, there are several reasons why it won't suit here.
I'd recommend to either use regular Table with checkboxes here or create some TenantEntity class working on top of Table's Row and providing appropriate methods.

Reply:
simply used regular `Table`